### PR TITLE
added a post

### PR DESCRIPTION
**Pull request by garyboygo**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!
  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

**Comment by lijiarui** [comment]
Please make CI green then I can review this PR.

**Comment by garyboygo** [comment]
@lijiarui Thanks for pointing that out for me! I already fix the test. Could you have me take a look at this PR when you have time?

**Comment by garyboygo** [comment]
Hi @lijiarui, Could you help me take a quick look at my PR when you have time?

File: jekyll/_developers/garyboygo.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Gary Peng
+site: https://github.com/garyboygo
+avatar: /jekyll/assets/developers/garyboygo/avatar.jpg
```
Comment (lijiarui) [inline jekyll/_developers/garyboygo.md]: Please change this to `/assets/developers/garyboygo/avatar.jpg`

And it seems you upload jepg, but you write jpg here.

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: "微信群广告检测机器人"
+author: gary
+categories: npm
+tags:
+  - wechaty
+  - wechat groups
+  - ads monitor
+image: /jekyll/assets/2020/ads-filter-groupbot/ads-filter-groupbot.jpeg
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-ads-filer-groupbot.md]: Please change to `/assets/2020/ads-filter-groupbot/ads-filter-groupbot.jpeg`

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- npm install wechaty@latest
+- npm install wechaty-puppet@latest
+- npm install wechaty-puppet-padplus@latest
+- npm install qrcode-terminal
+- npm install node-schedule
+- sudo npm install ts-node typescript -g
+
+## 最终效果
+
+![avatar](/jekyll/assets/2020/ads-filter-groupbot/ads-filter-groupbot.jpeg)
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-ads-filer-groupbot.md]: Please change to `/assets/2020/ads-filter-groupbot/ads-filter-groupbot.jpeg`

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: "微信群广告检测机器人"
+author: gary
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-ads-filer-groupbot.md]: Author name here should be same as the markdown file name. So you should change to `garyboygo` here.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: jekyll/_developers/garyboygo.md (hunk 1 above)
Comment (garyboygo) [reply jekyll/_developers/garyboygo.md]: Thanks for the suggestion!

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md (hunk 3 above)
Comment (garyboygo) [reply jekyll/_posts/2020-09-14-ads-filer-groupbot.md]: Done!

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md (hunk 4 above)
Comment (garyboygo) [reply jekyll/_posts/2020-09-14-ads-filer-groupbot.md]: Done!

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md (hunk 2 above)
Comment (garyboygo) [reply jekyll/_posts/2020-09-14-ads-filer-groupbot.md]: Done!

**Comment by garyboygo** [comment]
@lijiarui I already modified the code. Could you help me take a quick look at it?

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+## 功能
+
+- 检测用户发言是否有广告关键词
+- 识别出发了广告的用户并对其进行警告
+- 将因为发了广告被被踢出群的用户加入黑名单，之后不能再进入本群
+
+## 具体实现
+
+- bot.js: 项目入口文件，实现了扫码微信，并注册监测广告的逻辑。
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-ads-filer-groupbot.md:32]: Where is bot.js?

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2020-09-14-ads-filer-groupbot.md (hunk 5 above)
Comment (garyboygo) [reply jekyll/_posts/2020-09-14-ads-filer-groupbot.md:32]: bot.js is the entry point of the WeChatGroupBot. You can find it in here(https://github.com/garyboygo/WeChatGroupBot). And the utility.js provides a collections of utility function to bot.js

**Comment by huan** [comment]
Thank you very much for sharing this great post, the blog is online with URL <https://wechaty.js.org/2020/09/14/ads-filer-groupbot>

> You've invited garyboygo to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

1. I have invited you to join the Wechaty Contributor Team on our GitHub Wechaty Organization, please accept it by visiting <https://github.com/wechaty>
1. We have a WeChat room for Wechaty Contributors only, if you'd like to join, please contact @lijiarui on WeChat.

**Pull request merged**